### DISCUSSION TITLE
command line tool cfg file loading error

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#353] [Q&A]
I want to run the command line tool with a cfg file. 
By itself it runs through, however, I don't manage to load any cfg file. 
I get the following warning: 
![image](https://user-images.githubusercontent.com/98888664/159932039-686578ef-bd3c-4cc1-8e33-85e6348f3d28.png)

I have had similar issues with fasta and library before and you suggested that it were access rights issues. These could be solved by placing the relevant files on the home directory and it then worked. 
I have done the same for the cfg file. It doesn't work, neither from home nor from network directory. 

![image](https://user-images.githubusercontent.com/98888664/159932670-3d97b1c0-3b7d-4795-b228-26da16bca678.png)

the cfg file itself is very simple. It just says 
--matrices

**Comment by vdemichev** [comment]
The warning means that a simple std::ifstream contructor has failed, for whatever reason. Sorry, cannot help more with this.
